### PR TITLE
Add husky/linters/travis

### PR DESCRIPTION
**Pull request by vvscode**
Just basic stuff to keep code style consistent

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
 
 Structure of a repo:
 
 - `README.md` - this file
 - `types` - directory to keep all card types
+  Each card-type should be presented in form:
 
-Each card-type should be presented in form:
 1. Separate directory with card-type name
 1. `REAMDE.md` file wich explains the main purpose of type. Screenshots with examples and so on
```
Comment (zenby) [inline README.md]: Small typo - `REAMDE.md`

File: README.md [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
 1. `example.json5` - example card, described according to the card type
 1. `validator.js` - file, which should provide validation fuction, exported by default (in commonjs modules format), which should return `true/false` for incoming object. Better to use json-schema validation.
```
Comment (zenby) [inline README.md]: Small typo - `function`

File: package.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "card-types",
+  "version": "1.0.0",
+  "description": "Structure of a repo:",
+  "main": "index.js",
+  "scripts": {
+    "precommit": "lint-staged",
+    "lint:eslint": "eslint --quiet .",
+    "lint:json5": "json5 -v ./**/*.json5",
+    "lint:md": "prettier --list-different ./**/*.md",
```
Comment (zenby) [inline package.json]: Could you say, what is the role of this script if we dont use `markdown` now?

Comment (vvscode) [reply package.json]: It will be fore card-type description (there is a mention for `README.md` file )

File: README.md (hunk 2 above)
Comment (vvscode) [reply README.md]: thanks

File: README.md (hunk 1 above)
Comment (vvscode) [reply README.md]: fixed

**Pull request merged**